README.md

```diff
@@ -14,17 +14,138 @@ comes from a linked-data triple store via an API. ActiveRecord
 is not used for this application.
 
 The app does not interact directly with the triple store.
-Instead, we use a simple DSL to send queries to the
-[DS API](http://github.com/epimorphics/data-API). DS API uses
-the structure of the hypercube, represented as a _data structure
-definition_ (DSD), to convert the DSL queries to SPARQL.
+Instead, we use a JSON API wrapper based on SapiNT. The data API is not
+directly addressable from outside the deployment network, so to work with
+it in development we need to use SSH.
 
-### Accessing the API during development
+### Coding standards
+
+Rubocop should always return no errors or warnings.
 
-In deployment, the Rails app will be run in an environment in
-which the DS-API is available on `localhost:8080`. In development,
-the script `bin/sr-tunnel-daemon` can be used to simulate that
-environment, using ssh to proxy the remote dev API onto `localhost`.
+### Running the tests
+
+```sh
+rails -t
+```
+
+### Connecting to the data service
+
+The application needs to connect to a data API, based on
+[SapiNT](https://github.com/epimorphics/sapi-nt), to provide JSON-API access to
+the data in the triple store. The base URL for the data API needs to be passed
+as an environment variable `API_SERVICE_URL`. In production deployments, the
+`API_SERVICE_URL` will be set by Ansible configuration parameters.
+
+When developing PPD locally, the data API URL still needs to be passed via
+the environment. For example:
+
+```sh
+API_SERVICE_URL="http://....." rails server
+```
+
+To prevent unauthorised access to the API, the data API on deployment servers
+is protected so that it can only be accessed from `localhost`. Therefore, when
+working on API a developer needs to map access to the API so that it appears
+to be coming via localhost on the remote server. We can do this by creating
+an ssh tunnel.
+
+#### Setting up an ssh tunnel to access the data service
```

Comment:
Please explain why this is required?

Why not go via the load_balancer?

We should not popularizing the use of ssh tunnels to directly access the host.

Why are we duplicating the wiki? Create a link if needed. 
This is generic not to setup a ssh tunnel. The application readme is not the correct place for this even if it were a legitimate method. I wouldn't expect this to be duplicated in NRW applcations, and yet it is presumable just as valid.

Reply:
Thanks @andrew-pickin-epi. You've already given this feedback. We've got the message, there's really no need to keep repeating it. This is an old PR.

As I said on Slack, we should have a meeting to discuss the requirements that we have. I'd like to wait for everyone to be back around before we schedule that.